libcontainer/container.go

```diff
@@ -56,9 +57,12 @@ type BaseState struct {
 	// InitProcessPid is the init process id in the parent namespace.
 	InitProcessPid int `json:"init_process_pid"`
 
-	// InitProcessStartTime is the init process start time.
+	// InitProcessStartTime is the init process start time in clock cycles since boot time.
 	InitProcessStartTime string `json:"init_process_start"`
 
+	// CreationTimeStamp is the unix timestamp for the start time of the container in UTC
+	CreationTimeStamp time.Time `json:"creation_timestamp"`
```

Comment:
nit: for consistency we should remove the "Stamp" part - e.g. we don't have "InitProcessStartTime**Stamp**"